src/Integration.Vsix.UnitTests/SonarLintDaemon/SonarLintDaemonTests.cs

```diff
@@ -34,18 +34,19 @@ public class SonarLintDaemonTests
         private string tempPath;
         private string storagePath;
         private VSIX.SonarLintDaemon daemon;
+        private TestLogger logger;
 
         [TestInitialize]
         public void SetUp()
```

Comment:
I suggest to add tests for the "Verbose" logging - with and without "Server started" inside the `OutputDataReceived`

Reply:
Unfortunately, that can't be done as a unit test without substantial refactoring either - too risky this close to shipping.

The daemon code is also on my list to clean up, but it's not at the top :-(.